backend/plugins/frontline_api/src/modules/inbox/graphql/resolvers/queries/conversations.ts

```
@@ -20,8 +28,28 @@ export const conversationQueries = {
   async conversations(
     _parent: undefined,
     params: IConversationListParams,
-    { user, models, subdomain }: IContext,
+    { user, models, subdomain, serverTiming }: IContext,
   ) {
+    serverTiming?.startTime('conversations');
+
+    if (params && params.ids) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)